jupyterlitexblock/jupyterlitexblock.py

```diff
@@ -106,18 +109,23 @@ def render_template(self, template_path, context):
         rendered_template = template.render(Context(context))
         return rendered_template
 
-    def student_view(self, context=None):        
-        file_name = os.path.basename(self.default_notebook) if self.default_notebook else ''
+    def student_view(self, context=None):
+        file_name = os.path.basename(
+            self.default_notebook) if self.default_notebook else ''
         url = self.jupyterlite_url
         if file_name not in self.viewed_by_learner.split(','):
             self.viewed_by_learner += ',' + file_name
             url += f'?fromURL={self.default_notebook}'
         else:
             url += f'?path={file_name}'
-            
-        jupyterlite_iframe = '<iframe src="{}" width="100%" height="600px" style="border: none;"></iframe>'.format(url)
-        html = self.resource_string("static/html/jupyterlitexblock.html").format(jupyterlite_iframe=jupyterlite_iframe, self=self)
+
+        jupyterlite_iframe = '<iframe src="{}" width="100%" height="600px" style="border: none;"></iframe>'.format(
+            url)
+        html = self.resource_string("static/html/jupyterlitexblock.html").format(
```

Comment:
why you have span this statement over multiple lines?

Reply:
I have use auto pep8 formatter which caused this, the total characters io one line should be 79 according to pep 8 that's why formatter spanned it to multiple lines

Reply:
Wrong formatter issue